_posts/2021-03-18-Steering-Group-executive-summary,-action-items,-and-minutes.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Steering Group executive summary, action items, and minutes
+author: Franklin Feingold
+display: true
+---
+
+# Steering Group executive summary, action items, and minutes
+
+Date: Thursday March 18, 2021
+
+<!--more-->
+
+### Executive Summary
+
+The Steering Group spent the first part of this meeting discussing project sustainability. Then, we discussed our plans for OHBM. We’ll hold another Town Hall and use OHBM to broadcast our Steering Group election taking place in the Fall. 
+
+### Action items
+
+N/A
+
+### Minutes
+
+- PET validator extension is nearing completion
+- Project sustainability 
+- Potential Maintainer’s Group enhancement
+  - BEP representative that liaisons with the Maintainers
+- Use OHBM to do a callout for the next Steering Group member
+  - Elections are scheduled to take place this Fall
+- Enhancement to the website
+  - Rethink what we are trying to convey
+- OHBM: Hold BIDS Town Hall again this year in the Open Science Room
+- Process to depreciate tools
```

Comment:
Could you elaborate on what is meant by depreciation here, in particular for apps which aren't a part of bids project per se?

Reply:
This primarily refers to Apps that may not be active or maintained - creating a process to depreciate them. We want to convey active tools

Reply:
to add here: it is not about deprecating the tools (that is not for us to decide, but for the tool developer) but rather to curate the list on https://bids.neuroimaging.io/benefits.html to prevent that outdated, unsupported and deprecated tools obscure the ones that represent most user-value.